google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/ConnectionWorkerTest.java

```diff
@@ -279,26 +275,20 @@ public void testAppendInSameStream_switchSchema() throws Exception {
 
         // We will get the request as the pattern of:
         // (writer_stream: t1, schema: schema1)
-        // (writer_stream: _, schema: _)
+        // (writer_stream: t1, schema: _)
```

Comment:
How do we know we are multiplexed before the actual switch?

Reply:
the connection would be created via connectionPool